electron-app/tests/e2e/pages/account-balances-page.ts

```diff
@@ -1,7 +1,9 @@
 // import { capitalize } from '../../../src/filters';
+import { capitalize } from '@/filters';
+
 import { ApiManualBalance } from '../../../src/services/types-api';
 import { bigNumberify, Zero } from '../../../src/utils/bignumbers';
-import { capitalize } from '../../common/helpers';
+// import { capitalize } from '../../common/helpers';
```

Comment:
I am guessing we don't need that right?

Reply:
I have it deleted just forgot to save the file =P